.github/workflows/main.yaml

```diff
@@ -72,7 +72,7 @@ env:
   QASE_REPORT: 1
   QASE_RUN_COMPLETE: 1
   RANCHER_LOG_COLLECTOR: ${{ github.workspace }}/.github/scripts/collect-rancher-logs.sh
-  GCP_RUNNER_ZONE: asia-south2-c
+  GCP_RUNNER_ZONE: us-west1-c
```

Comment:
You don't need to change the runner template, do you? You mentioned something about copying it from asia-south2-c to us-west1-c?